testdata/7/code.go

```diff
@@ -4,28 +4,28 @@ import (
 	"errors"
 )
 
-// Those are not errors
-var myVar = 1
-var errFakeErrorUnexported = 1
-var ErrFakeErrorExported = 1
-
-// Those errors are not named correctly
-var myErrVar = errors.New("myErrVar")
-var myVarErr = errors.New("myVarErr")
-var myVarError = errors.New("myVarErr")
-
-// var customErr = customError{"customErr"}
-
-// Those errors have a declared error type
-var declaredErr error = errors.New("declaredErr")
-var errDeclared error = errors.New("errDeclared")
-
-// Those are actual errors which should be ignored
-var errUnexported = errors.New("errUnexported")
-var ErrExported = errors.New("ErrExported")
-
-// var errCustomUnexported = customError{"errCustomUnexported"}
-// var ErrCustomExported = customError{"ErrCustomExported"}
+var (
+	// Those are not errors
+	myVar                  = 1
+	errFakeErrorUnexported = 1
+	ErrFakeErrorExported   = 1
+
+	// Those errors are not named correctly
+	myErrVar   = errors.New("myErrVar")
+	myVarErr   = errors.New("myVarErr")
+	myVarError = errors.New("myVarErr")
+	// customErr = customError{"customErr"}
+
+	// Those errors have a declared error type
+	declaredErr error = errors.New("declaredErr")
+	errDeclared error = errors.New("errDeclared")
+
+	// Those are actual errors which should be ignored
+	errUnexported = errors.New("errUnexported")
+	ErrExported   = errors.New("ErrExported")
+	// errCustomUnexported = customError{"errCustomUnexported"}
+	// ErrCustomExported   = customError{"ErrCustomExported"}
+)
```

Comment:
🎉 This is great. Can we make this `testdata/8` so that we have both top level and grouped vars being tested of this kind?

Reply:
Shall I simply `cp -R testdata{7,8}` and make `testdata8` use grouped vars?

Reply:
Yup I think that'd be perfect.